lib/gold_miner/slack/client.rb

```diff
@@ -63,20 +64,25 @@ def extract_messages_from_result(result)
       result.messages.matches.map do |match|
         Slack::Message.new(
           text: match.text,
-          author: fetch_author(match),
+          author: build_author(match),
           permalink: match.permalink
         )
       end
     end
 
-    def fetch_author(message)
+    def build_author(message)
       Slack::User.new(
```

Comment:
I'd love to call this class `Author`, instead of `Slack::User`. Maybe there are two objects here?

1. `Slack::User`: contains:
    - `id` (Slack id) 
    - `username` (Slack username)
2. `BlogPost::Author`: contains:
    - `name` (which I'm planning to pull from a config instead of only using what's on Slack)
    - `link`: preferred link to use

And the `BlogPost::Author` might receive a `Slack::User` on its constructor.

Thoughts?